atlas/atlas.star

```diff
@@ -0,0 +1,175 @@
+"""
+A URL is an absolute location within the atlas.
+"""
+scalar URL
+
+"""
+Names are dotted paths of GraphQL identifiers, like `spotify.User`
+and `String`.
+"""
+scalar Name
+
+type Query {
+  at(id: URL): Point
+  edge(name: Name, from: Name, to: Name): Edge
+  type(id: URL): Type
+  interface(id: URL): Interface
+  edge(id: URL): Edge    
```

Comment:
Since we have a `Point` interface, instead can we have a `point(id: URL)` and `points(ids: [URL!]!)` here instead of one off for each type of point in the graph?